mu-plugins/central-hub/src/config-store/internals.php

```diff
@@ -62,13 +62,22 @@ function _the_store( $store_key = '', $config_to_store = array(), $remove = null
  * Load a configuration from the filesystem, returning its
  * storage key and configuration parameters.
  *
- * @since 1.0.0
- *
  * @param string $path_to_file Absolute path to the config file.
  *
  * @return array
+ * @throws \Exception
+ * @since 1.1.0
+ *
  */
 function _load_config_from_filesystem( $path_to_file ) {
+	if ( ! $path_to_file ) {
+		throw new \Exception(
+			sprintf(
+				'The configuration is empty and the store_key does not exist.'
+			)
+		);
+	}
+
 	$config = (array) require $path_to_file;
 
 	return array(
```

Comment:
Let's revisit what you are trying to check for:

1. Empty key means no store key.
2. Empty config means nothing to store in the store.

How can we check that?  Here's one implementation:

```php
/**
 * Load a configuration from the filesystem, returning its storage key and configuration parameters.
 *
 * @since 1.0.0
 *
 * @param string $path_to_file Absolute path to the config file.
 *
 * @return array returns an array with storage key => config parameters.
 * @throws \Exception
 */
function _load_config_from_filesystem( $path_to_file ) {
	$config    = (array) require $path_to_file;

	$store_key = key( $config );
	if ( empty( $store_key ) ) {
		throw new \Exception(
			sprintf( 'No store key exists in the %s configuration file.', esc_attr( $path_to_file ) )
		);
	}

	$config_params = current( $config );
	if ( empty( $config_params ) ) {
		throw new \Exception(
			sprintf( 'No configuration parameters exist for store key [%s] in the %s configuration file.',
				esc_attr( $store_key ), esc_attr( $path_to_file ) )
		);
	}

	return [ $store_key, $config_params ];
}
```

Reply:
@hellofromtonya Elegant and practical! Also, you clarified something for me by this example. 

I was unsure whether to add a string placeholder ( [%s] ) to the Exception message. You just answered my question with your example.

Reply:
@hellofromtonya Code above incorporated into branch.